lib/features/tabs/ui/screens/components/top_picks_section_header.dart

```diff
@@ -0,0 +1,25 @@
+import 'package:elvan/shared/components/constants/app_colors.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter/src/widgets/container.dart';
+import 'package:flutter/src/widgets/framework.dart';
+
+class TopPicksSectionHeader extends StatelessWidget {
+  const TopPicksSectionHeader({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return Row(
+      mainAxisAlignment: MainAxisAlignment.spaceBetween,
+      children: const [
+        Text(
+          'Top Picks',
+          style: TextStyle(
+              fontSize: 28,
```

Comment:
Use Material 3 sizing from Theme object like -> Theme.of(context).textTheme.bodyLarge. Look at Figma file to know which component uses what material text.